proto/greenfield/storage/query.proto

```diff
@@ -356,4 +370,27 @@ message QueryIsPriceChangedResponse {
     (gogoproto.nullable) = false
   ];
 }
+
+message QueryGroupMembersExistRequest {
+  string group_id = 1;
+  repeated string members = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+message QueryGroupMembersExistResponse {
+  repeated bool are_members = 1;
+}
+
+message QueryGroupsExistRequest {
+  repeated string group_names = 1;
+  repeated string group_owners = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+message QueryGroupsExistByIdRequest {
+  repeated string group_ids = 1;
+}
+
+message QueryGroupsExistResponse {
```

Comment:
Shall we unify the name of the API? E,g. `HeadGroup`, `HeadGroups`, `HeadGroupById`, `HeadGroupsByIds`, `HeadGroupMember`, `HeanGroupMemebers`

Reply:
OK